pkgs/development/zig-packages/build-zig-package/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, zig }:
+
+{ strictDeps ? true
+, nativeBuildInputs ? [ ]
+, enableParallelBuilding ? true
+, meta ? { }
+, ...
+}@attrs:
+
+stdenv.mkDerivation (attrs // {
```

Comment:
In this specific line, I would like the help of @roberth. 
As the creator of the new `rec`-less `mkDerivation` style (https://github.com/NixOS/nixpkgs/pull/119942), do you believe this function can be made in a similar way?